File: Assignment No_16/Question_4.c
/*4.Accept N numbers from user and accept Range,Display all elements from that range.

Input : N : 6
        Start: 60
        End  : 90
        Elements :85 66 3 76 93 88

Output : 66 76 88

*/
////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

#include<stdio.h>
#include<stdlib.h>

int RangeDisplay(int Arr[], int iSize, int iStart, int iEnd)
{
  int iCnt=0, iAns=0;

  for(iCnt=0; iCnt<iSize; iCnt++)
  {
    if((Arr[iCnt] >=iStart) && (Arr[iCnt] <= iEnd) )
      printf("%d\t",Arr[iCnt]);
    
  }

 
}

int main()
{
   int iSize=0, iCnt=0, iValue1=0, iValue2=0;
   int *Array=NULL;
   int iRet=0;

   printf("Enter the Number of Elements:\t");
   scanf("%d",&iSize);

   printf("Enter the Starting point");
   scanf("%d",&iValue1);

   printf("Enter the Ending point");
   scanf("%d",&iValue2);

   Array=(int *)malloc(iSize*sizeof(int));

   if(Array==NULL)
     return -1;

   printf("Enter %d Elements",iSize);
   for(iCnt=0; iCnt<iSize; iCnt++)
   {
     scanf("%d",&Array[iCnt]);
   }

   RangeDisplay(Array,iSize,iValue1, iValue2);
  
   free(Array);

   return 0;
}

File: Assignment No_39/Question_4.java
 /* 4. Write java program which accept N numbers from user and accept Range, Display all elements from that range

Input : N : 6
        Start: 60
        End : 90
Elements :85 66 3 76 93 88
Output : 66 76 88

Input : N : 6
        Start: 30
        End : 50
Elements :85 66 3 76 93 88
Output :

*/
//////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

 
 import java.util.*;

class Ass39_4
{
    public static void Display_Range(int Brr[], int iStart, int iEnd)
    {
        int i = 0;

        for(i = 0; i < Brr.length; i++)
        {
            if((Brr[i] > iStart) && (Brr[i] < iEnd))
            {
                System.out.println(+Brr[i]);
            }
        }

        
    }

    public static void main(String Arg[])
    {
        Scanner sobj = new Scanner(System.in); 
        int i = 0, iSize = 0, iStart = 0, iEnd=0;
        int iRet = 0;

        System.out.println("Enter the number of elements : ");
        iSize = sobj.nextInt();

        int Arr[] = new int[iSize];

        System.out.println("Enter the values : ");

        for(i = 0; i < iSize; i++)
        {
            Arr[i] = sobj.nextInt();
        }

        System.out.println("Elements of the array are : ");

        for(i = 0; i < iSize; i++)
        {
            System.out.print(Arr[i]+"\t");
        }
        System.out.println();

        System.out.println("Enter the Starting Point : ");
        iStart = sobj.nextInt();

        System.out.println("Enter the Ending Point: ");
        iEnd = sobj.nextInt();

        Display_Range(Arr,iStart, iEnd);

    }
}





